src/app/models/indicator-params.model.ts

```diff
@@ -0,0 +1,10 @@
+import { ClimateModel } from './climate-model.model';
```

Comment:
Now there are two models:
indicator-parameters.model.ts
indicator-params.model.ts

in the models folder, and that is confusing

Reply:
Any suggestions for better names? otherwise I'll try to come up with something

Reply:
We've been using the term "extra params"
Maybe 
indicator-extra-params?

Reply:
Or perhaps indicator-query-parameters

Reply:
We already have indicator-query-opts and these params aren't always the subset that are the extra params. They're really the GET params to an API request. What about the following refactor:
- `IndicatorQueryOpts -> IndicatorRequestOpts`
- `IndicatorParams -> IndicatorGetParams`
- Leave `IndicatorParam` unchanged

Reply:
OK IndicatorRequestOpts
#2 -- > IndicatorParamsList? or IndicatorQueryStringParams?
OK IndicatorParam

Reply:
`IndicatorQueryParams` would just be a shorter version of your second option that isn't any less descriptive. How about that? Definitely like your second suggestion better than the first.